AppController/lib/zkinterface.rb

```diff
@@ -501,6 +501,15 @@ def self.get_version_details(project_id, service_id, version_id)
   end
 
 
+  # Defines a deployment-wide default for maximum instance memory.
+  def self.set_max_memory(max_memory)
+    max_memory_node = '/appscale/config/max_memory'
+    data = {'max_memory' => max_memory}
```

Comment:
Isn't it better to save just integer number instead of object? `"max_memory"` is already presented in node path.